DXRModules/DeusEx/Classes/DXRLoadouts.uc

```diff
@@ -120,7 +120,7 @@ function CheckConfig()
         item_sets[5].allows = "WeaponCrowbar";
         item_sets[5].starting_equipments = "WeaponCrowbar";
 
-        item_sets[6].name = "Grenades Only";
+        item_sets[6].name = "Grenades Only!";
```

Comment:
massive improvement here! did you mean to add this to explosives only? lol

Reply:
I'm a really big consistency fan.

Reply:
but most of the other names don't have it?

Reply:
also I'm wondering if we should add rubber baton to a bunch of the other loadouts

Reply:
Oh, I meant to add it to the player_message not the name. The other flavorful denial messages (as opposed to eg "No Pistols") mostly all end in exclamation marks.

Reply:
> also I'm wondering if we should add rubber baton to a bunch of the other loadouts

Could it be made even weaker somehow, like changing its damage to zero if the target is a Pawn? I was knocking out UNATCO guys in M02 in one hit earlier.

Reply:
sort of, there's technically not an easy way to query what the target is, for melee weapons you can kind of reliably use the highlight target but idk if I want to rely on that

I guess if it accidentally does 1 damage instead of 0 then it's not a big deal